client/components/checklist/style.scss

```diff
@@ -11,7 +15,7 @@
 		padding: 0;
 	}
 
-	&-main {
+	.checklist__header-main {
```

Comment:
Curious, why the change to spelling out the classname explicitly? To make it easier to `grep`?

Reply:
I also prefer full explicit class names for clarity (just how my brain works) as well for easier searching. No strong opinion the way or the other tho. :-)

Reply:
Yeah, I did it for searching plus it's also easier to read.